src/main/java/org/zendesk/client/v2/model/Action.java

```diff
@@ -32,11 +32,11 @@ public void setField(String field) {
         this.field = field;
     }
 
-    public String[] getValue() {
+    public Object[] getValue() {
         return value;
     }
 
-    public void setValue(String[] value) {
+    public void setValue(Object[] value) {
```

Comment:
The internal structure change is OK.
The problem, I believe, is changing the method signatures. The clients compiled against the older versions won't be able to use this new version because of class casting or method-binding issues if I understand correctly. 
This, therefore, might need a major version bump.

Reply:
In the past I used Clirr ( http://www.mojohaus.org/clirr-maven-plugin/ ) but I don't think it was updated to manage recent java versions. But for sure such tools are existing @duemir @Dohbedoh

Reply:
Right I think this breaks compatibility. Although the signature change to a broader class... I am not sure at the moment how we could avoid this to fix that bug. Only solution I could think of is to use a custom Jackson serializer.

Reply:
Arrays are covariant in Java so the setter is probably not an issue.
The getter is what worries me since in the client's code there'll be an assignment to a `String[]` but the new code returns `Object[]`

Reply:
I remember reading a solution for this kind of issue used by netbeans consisting of patching the bytecode